### DISCUSSION TITLE
Erroneous Genomes

### DISCUSSION THREAD

**Discussion by RyanCook94** [RyanCook94/inphared#1] [General]
Because of the automated nature of the script, some erroneous non-phage genomes may slip through. Please let me know if you spot any and I'll add them to the exclusion list!

**Comment by amillard** [comment]
LOCUS       MG065691              174114 bp    DNA     linear   PHG 01-JAN-2018
DEFINITION  UNVERIFIED: Campylobacter phage A11a, complete genome.         This is mostly likely not a phage  so can be added to exclusion list. It is 100% identical to E.coli genome (not a prophage region as far as can tell)

**Comment by amillard** [comment]
This has slipped through MZ089735 and needs to be added to the exclusion list

**Comment by amillard** [comment]
@ryan . I have just found ~500 complete or complete coding genomes via ICTV that we should add to the inclusion list, that our criteria miss. 

BK014188
BK013603
BK014009
BK013977
BK013744
BK013754
KT462700
KT462701
BK013715
BK013911
BK013882
BK013443
BK014194
BK013654
BK013891
BK013480
BK013490
BK013910
BK014035
BK013987
BK013827
BK014032
BK014031
BK013870
BK013499
BK013511
BK013792
BK013396
BK014165
BK013591
BK013689
BK013379
BK013690
BK014142
BK013413
BK014001
BK013454
BK013452
BK014018
BK013787
BK014000
BK013951
BK013534
BK013927
BK013848
BK013455
BK013831
BK013906
BK014119
BK013639
BK014120
BK013737
BK014122
BK014189
BK013485
BK013473
BK014083
BK014028
BK013533
BK013980
BK014023
BK013747
BK013772
BK013806
BK013748
BK013821
BK013678
BK013438
BK013925
BK013945
BK013666
BK013923
BK013439
BK014118
BK013919
BK014176
BK013820
BK014027
BK013686
BK013523
BK013786
BK013570
BK013931
BK013989
BK013947
BK013811
BK013561
BK013407
BK013889
BK013477
BK013546
BK013553
BK013969
BK013926
BK013466
BK013917
BK013790
BK013540
BK013447
BK013384
BK013825
BK013733
BK013595
BK014137
BK013668
BK014090
BK013988
BK013419
BK014121
BK014041
BK013572
BK013920
BK014014
BK013844
BK013858
BK013970
BK013691
BK014139
BK014133
BK013890
BK013484
BK013512
BK013516
BK013809
BK014043
BK013954
BK013851
BK014007
BK014024
BK014193
BK013712
BK013599
BK014144
BK013486
BK013857
BK014030
BK013563
BK013694
BK014064
BK013446
BK013538
BK013864
BK014114
BK013641
BK014152
BK013968
BK013872
BK013389
BK013878
BK013943
BK013993
BK013914
BK013671
BK013596
BK013734
BK014080
BK014196
BK013465
BK013834
BK013887
BK013476
BK013830
BK013932
BK013481
BK013937
BK013567
BK013423
BK014216
BK014161
BK014199
BK013658
BK013601
BK013798
BK013897
BK013659
BK013963
BK013426
BK013950
BK013953
BK013424
BK013871
BK013769
BK013557
BK013504
BK013488
BK013467
BK013556
BK013836
BK013441
BK013510
BK013552
BK013385
BK013527
BK014026
BK013955
BK014148
BK013985
BK013964
BK013451
BK014138
BK013801
BK013894
BK014184
BK014178
BK013588
BK014179
BK014048
BK013578
BK013997
BK013393
BK013942
BK013470
BK013500
BK013837
BK013394
BK013541
BK013939
BK013402
BK013568
BK013459
BK013881
BK013559
BK013999
BK013457
BK013550
BK013430
BK013507
BK013829
BK013410
BK013506
BK013566
BK013879
BK013400
BK013479
BK014005
BK013867
BK013404
BK013381
BK013501
BK013828
BK013794
BK013974
BK013814
BK014004
BK013817
BK013391
BK013388
BK013383
BK013560
BK013775
BK013450
BK013884
BK013813
BK013930
BK013796
BK013432
BK013991
BK013940
BK013962
BK013965
BK013408
BK013468
BK013779
BK013558
BK013449
BK013956
BK013976
BK013909
BK013458
BK014029
BK014042
BK013896
BK013803
BK013982
BK013482
BK013461
BK013928
BK013397
BK013839
BK013866
BK013411
BK013973
BK013808
BK013386
BK013895
BK013845
BK014022
BK014015
BK014033
BK013494
BK013832
BK013417
BK013409
BK013960
BK013495
BK013493
BK013705
BK013924
BK013435
BK014045
BK013535
BK013565
BK014012
BK013903
BK013816
BK013463
BK013804
BK013843
BK014010
BK013632
BK013998
BK013826
BK013600
BK013399
BK014084
BK013704
BK014071
BK014135
BK014168
BK014091
BK013944
BK013756
BK013873
BK014213
BK013571
BK013789
BK013412
BK013979
BK013642
BK013853
BK013763
BK013938
BK013886
BK013908
BK013774
BK013474
BK014019
BK013371
BK013542
BK013425
BK013475
BK013401
BK013576
BK013418
BK013776
BK014044
BK013865
BK014102
BK013992
BK013652
BK014210
BK014131
BK013647
BK014108
BK013723
BK013771
BK013517
BK014008
BK013431
BK013941
BK013536
BK013835
BK013984
BK014020
BK013532
BK013815
BK013520
BK013496
BK013805
BK013971
BK013905
BK013731
BK013589
BK013648
BK013841
BK013529
BK013922
BK013530
BK013509
BK013788
BK013370
BK013525
BK013508
BK013810
BK013868
BK013544
BK013797
BK014187
BK013978
BK013850
BK013874
BK013387
BK013860
BK013912
BK014034
BK013994
BK014013
BK013842
BK013898
BK013907
BK014126
BK013515
BK013869
BK013782
BK013398
BK013904
BK014200
BK013670
BK014117
BK014107
BK013861
BK013555
BK013537
BK014025
BK014065
BK013952
BK014038
BK013818
BK013543
BK014124
BK013587
BK014163
BK013586
BK013380
BK013740
BK013590
BK013584
BK013518
BK013681
BK014181
BK013634
BK013460
BK013428
BK013633
BK013876
BK013359
BK013703
BK014162
BK013730
BK013675
BK013657
BK013742
BK014059
BK014053
BK013741
BK013360
BK013575
BK013362
KT462698
BK013638
BK014052
BK013717
BK013422
BK014097
BK013364
BK013372
BK013693
BK013680
BK014076
BK014057
BK013478
BK013966
BK013579
BK014109
BK014074
BK013663
BK014055
BK014159
BK014154
BK013604
BK014078
BK013795
BK013444
BK013793
BK013462
BK014169
BK013621
BK013673
BK013710
BK014141
BK014167
BK014130
BK013709
BK014166
BK013716
BK013644
BK014003
BK013746
BK013738
BK014125
BK013750
BK013762
BK014204
BK013627
BK014145
BK014150
BK013367
BK013645
BK013777
BK014021
BK013472
BK014006
BK013735
BK013582
BK013687
BK013656
BK014218
BK014201
BK013852
BK013736
BK013757
BK014087
BK013683
BK013649
BK013958
BK013390
BK013892
BK013610
BK014103
BK013729
BK013382
BK013718
BK013913
BK013791
BK013933
BK013732
BK013724
BK013698
BK013573
BK014073
BK014068
BK014058
BK014183
BK013613
BK014143
BK014086
BK014202
BK013660
BK013725
BK013577
BK013807
BK013719
BK014147
BK014132
BK014082
BK013415
BK013497
BK013406
BK013833
BK013959
BK014075
BK013800
BK014217
BK013551
BK013505
BK013519
BK013946
BK013513
BK013883
BK013355
BK013672
BK013916
BK014051
BK013822
BK013990
BK013503
BK013759
BK013608
BK013662
BK014115
BK013592
BK014113
BK013366
BK013487
BK013464
BK013436
BK013961
BK014077
LT821717
BK013679
BK013651
BK013628
BK014128
BK013623
BK013996
BK013569
BK013489
BK013983
BK013824
BK013491
BK014170
BK013356
BK014172
BK014197
BK013614
BK013626
BK014191
BK013605
BK013395
BK013863
BK014089
BK014085
BK014190
BK013665
BK013812
BK013799
BK013612
BK014160
BK013593
BK013655
BK014070
BK014066
BK013521
BK013899
BK013445
BK013838
BK013893
BK013606
BK014208
BK014164
BK014099
BK013745
BK014177
BK013531
BK013986
BK013434
BK014180
BK013706
BK013674
BK013854
BK013900
BK013676
BK014049
BK013957
BK013374
BK013637
BK013699
BK013784
BK014105
BK013753
BK013682
BK013846
BK013427
BK013819
BK013783
BK014158
BK014101
BK014063
BK013646
BK014146
BK013376
BK014116
BK014061
BK013617
BK013598
BK013581
BK013619
BK014195
BK014054
BK014046
BK014060
BK014095
BK013700
BK013378
BK014198
BK013669
BK014215
BK013758
BK014207
BK013615
BK013721
BK013767
BK014205
BK013585
BK014056
BK013429
BK013616
BK013739
BK014096
BK013405
BK014079
BK013877
BK013847
BK013677
BK014088
BK013702
BK013692
BK013640
BK014050
BK013363
BK013752
BK013361
BK013369
BK013802
BK013622
BK013849
BK013921
BK014155
BK013554
BK013743
BK013635
BK013643
BK013625
BK013618
BK013358
BK014067
BK013365
BK013901
BK013433
BK013528
BK013823
BK014036
BK013859
BK013778
BK013972
BK013856
BK014016
BK013650
BK014221
BK013722
BK013594
BK014174
BK014106
BK013620
BK013929
BK013442
BK013547
BK013357
BK013580
BK013708
BK013707
BK013636
BK013995
BK013437
BK014040
BK014111
BK013936
BK013697
BK013453
BK013440
BK013377
BK013714
BK014069
BK014062
BK013597
BK013667
BK014134
BK013624
BK013602
BK013701
BK013696
BK013583
BK014153
BK014186
BK014182
BK014220
BK014047
BK013720
BK014175
BK014157
BK013548
BK014100
BK013766
BK013688
BK014127
BK014037
BK013483
BK013549
BK013918
BK013773
BK013749
BK013862
BK013471
BK013368
BK014173
BK014092
BK013685
BK013611
BK013751
BK014219
BK013949
BK013492
BK013403
BK014104
BK014149
BK014209
BK014094
BK013761
BK013416
BK013967
BK013607
BK014017
BK013781
BK013456
BK013526
BK013514
BK013564
BK014002
BK014185
BK014156
BK013661
BK014093
BK014214
BK013764
BK014203
BK014151
BK013629
BK013373
BK013770
BK014011
BK013935
BK013948
BK014039
BK013902
BK013975
BK014212
BK013765
BK013726
BK014072
BK013880
BK014171
BK013711
BK013840
BK013855
BK013915
BK013414
BK013785
BK013885
BK014206
BK013695
BK013664
BK013630
BK014123
BK014129
BK013755
BK014211
BK013609
BK014081
BK013420
BK013684
BK013768
BK013981
BK013888
BK013502
BK013875
BK013934
BK013392
BK013524
BK014112
BK013728
BK013574
BK013448
BK013760
BK013653
BK013469
BK013780
BK013522
BK014140
BK014136
BK013545
BK013727
BK013539
BK013562
BK014110
BK013375
BK014098
BK013631
BK013713
BK014192
BK013498
BK013421
MF361639
MG592455
MG592402
MG592394
MG592427
MG592615
JQ995537
MH616963
MH552500
AP013538
AP013544
KT997847
AP013545
AP013543
GU943031
KX397280
AP013549
HQ633062
AP013547
AP013546
GU943068
AP013539
GU943054
AP013542
AP013541
AP013548
AP013540
AC171169
AY576273
MF417929
KK213166
AE006468
AY319521
CP038625
LR797314
CP023680
CP023686
CP006891
CP014526
CP019275
CP052639
HM208303
MW175890
GU339467
EF489910
AM040673
MN335248
HM066936
AF059242
DQ836063
JX045650
JX625144
JX045649
AF334111

**Comment by SMichniewski** [comment]
Few that need double checking before exclusion:
AGRM01000001-AGRM01000007 - bacterial WGS (BCT)
AFYI01000001-AFYI01000008 - bacterial WGS (BCT)
ACSJ01000001-ACSJ01000019 - bacterial WGS (BCT)
AB571791 - genomic island, BCT
CAKLQH020000001-CAKLQH020000123 - supposedly phage WGS, looks like bacterial WGS though (120+ contigs, total >3mb data) with some prophage (or phage) contigs included
CAKLQF020000001-CAKLQF020000085 - as above

**Reply by RyanCook94** [reply]
Cheers for flagging these! Will add to the exclusion list. Not sure how a couple of those crept through...

**Comment by QLamyBesnier** [comment]
Hello, I've noticed 4 phages with highly suspicious genomes : NC_069752, NC_069753, NC_069754, NC_069755.
They are clones of the same "PSV3" phage according to their name, and all have the same issue of having many duplicated genes. Other phages of the same genus "Septimatrevirus" did not have such duplications.
Double-checking is welcome but in my opinion those are not proper phage genomes and should be excluded from INPHARED.

**Reply by amillard** [reply]
Interesting, these are all REFSEQ accession numbers as well.  Thank you for flagging , will have a look at them.
